css-three-column-layout/styles.css

```diff
@@ -0,0 +1,56 @@
+* {
+  background-color: rgba(25, 150, 218, 255);
+  box-sizing: border-box;
+  font-family: "lato", sans-serif;
+  color: #fff;
+}
+
+.container {
+  max-width: 1000px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.column-third {
+  float: left;
```

Comment:
Instead of using `float` here, you should make a `row` class with the styling `display: flex`, which will force your items onto one line. Make sure you make all three `.column-third` elements a child of the element with the `row` class 🤓